### ISSUE TITLE
Logs component from a plugin results in blank page

### ISSUE THREAD

**Issue by ImJasonH** [vmware-tanzu/octant#307]
I have code like this in my plugin:

```go
func handlePrint(request *service.PrintRequest) (plugin.PrintResponse, error) {
  resp := plugin.PrintResponse{}
  logCard := component.NewCard("Logs")
  logCard.SetBody(component.NewLogs("tekton-pipelines",
    "tekton-pipelines-controller-5f6d47fdd9-snlnm",
    []string{"tekton-pipelines-controller"}))
  resp.Items = append(resp.Items, component.FlexLayoutItem{
    Width: component.WidthFull,
    View:  logCard,
  })
  return resp
}
```

(This pod with this container is running in this namespace, and works just fine in the Pod's Logs tab)

This causes the page for this resource to be rendered blank, with no errors in the Chrome Console or in `octant`'s output.

**Comment by bryanl** [comment]
This is interesting. This is definitely not a scenario I envisioned when putting this together. Will take a look.

**Comment by ImJasonH** [comment]
Hey, has there been any change here? I'd love to be able to show Pod logs in my plugin component, but I realize this might not be a high priority for anybody else.

**Comment by wwitzel3** [comment]
It has not been looked at yet, since this is something that is actively holding you up, I'll bump it in to the sprint backlog so we can investigate it more.

**Comment by wwitzel3** [comment]
fixed with #618
